docs/sources/next/javascript-api/k6-net-grpc/stream/message-metadata.md

```diff
@@ -0,0 +1,48 @@
+---
+title: 'Message Metadata'
+description: 'An object with the metadata of a message from gRPC stream.'
+weight: 50
+---
+
+# Message Metadata
```

Comment:
```suggestion
title: 'Metadata'
description: 'The metadata of a gRPC stream's message.'
weight: 50
---

# Metadata
```

Reply:
I've applied this locally since the `'` inside the description text breaks the YAML